msprime/simulations.py

```diff
@@ -1176,8 +1176,38 @@ class SmcPrimeApproxCoalescent(SimulationModel):
 
 class DiscreteTimeWrightFisher(SimulationModel):
     """
-    A discrete backwards-time Wright Fisher model, with back-and-forth
-    recombination
+    A discrete backwards-time Wright-Fisher model, with diploid back-and-forth
+    recombination.
+
+    Wright-Fisher simulations are performed very similarly to coalescent
+    simulations, with all parameters denoting the same quantities in both
+    models. Because events occur at discrete times however, the order in which
+    they occur matters. Each generation consists of the following ordered
+    events:
+
+    1) Migration events. As in the Hudson coalescent, these move single extant
+    lineages between populations. Because migration events occur before
+    lineages choose parents, migrant lineages choose parents from their new
+    population in the same generation. Nodes, including sample nodes, are never
```

Comment:
I don't follow this last sentence - perhaps just delete it?

Reply:
Ya agreed, I don't think it's necessary